scripts/svm/createVault.ts

```diff
@@ -0,0 +1,66 @@
+// This script is used by a chain admin to create a vault for a token on the Solana Spoke Pool.
```

Comment:
this can, in theory, be called by anyone other than just the admin, right? as we are simply creating a ATA for the spokepool PDA to store a given token?

Im thinking through the equivalent functionality we have on other chains now that we no longer have a deposit route check: in theory, someone else could call use this script (or otherwise) to create PDAs so that you can deposit and fill non-whitelisted tokens on behalf of the protocol, just like we can on other chains (by just depositing without using a verified token to a given spoke). 

point being, this is not used by an admin, right? there is no special permissioning to create one of these vaults?

Reply:
Indeed, anyone can create the vault. normally we expect admin to do this as that requires spending rent-exempt deposit on vault ATA creation. But any other user is free to do this if they want to deposit funds on a non-whitelisted token.

Reply:
updated the comment to reflect that